hw2/03-count.js

```diff
@@ -1,3 +1,25 @@
 // Add your code here
 
-input.addEventListener('keydown', handleKeyDown);
+document.addEventListener("DOMContentLoaded", () => {
+    const wordInput = document.getElementById("wordInput");
+    const textBox = document.querySelector(".textBox");
+  
+    wordInput.addEventListener("input", () => {
+      const searchWord = wordInput.value.trim();
+      const text = textBox.innerText;
+      const words = text.split(/\s+/);
+  
+      let highlightedText = "";
+      words.forEach((word) => {
+        const regex = new RegExp(`\\b${searchWord}\\b`, "i");
+        if (regex.test(word)) {
+          highlightedText += `<span class="highlight">${word}</span> `;
+        } 
+        else {
+          highlightedText += `${word} `;
+        }
+      });
```

Comment:
At the moment, when you clear out the wordInput, it highlights the whole passage. I would nest this in a if statement to check if the wordInput is empty or not. I think you would also need to keep an original version of the passage to restore in this case as well.

Reply:
Completed! Thanks for your review!